File: src/Paiza/python/D071.py
# 세탁물과 먼지
# 파이썬에 논리 연산자에 대해 공부했다.
temp, opreate = map(int, input().split())

if temp >= 25 and opreate <= 60:
    print("Yes")
elif opreate <= 20:
    print("Yes")
else:
    print("No")

File: src/Paiza/D071.kt
package Paiza

//세탁물과 먼지
//일본어 해석이 이상해서 문제를 재대로 모르겠다.
fun main() {
    val temp = readLine()!!.split(" ").map { it.toInt() }

    if (temp[0] >= 25 && temp[1] <= 60){
        println("Yes")
    }else if (temp[1] <= 20){
        println("Yes")
    }else{
        println("No")
    }

}